### PR TITLE
separate input and output of fillBCTest() function

### PR DESCRIPTION
**Pull request by pirapira**
This is the first step for separating input and output of doBlockchainTest() function.

This PR changes `fillBCTest()` function so that it takes an input and returns an output.  Before this PR, the function modified a filler into a completed test in place.

### CODE REVIEW

File: test/tools/jsontests/BlockChainTests.cpp [hunk 1]
```diff
@@ -244,46 +244,65 @@ json_spirit::mValue doBlockchainTestNoLog(json_spirit::mValue const& _input, boo
 	return v;
 }
 
-void fillBCTest(json_spirit::mObject& _o)
+json_spirit::mObject fillBCTest(json_spirit::mObject const& _input)
 {
+	json_spirit::mObject output;
```
Comment (winsvega) [inline test/tools/jsontests/BlockChainTests.cpp]: could you name it outputJson?

**Comment by pirapira** [comment]
I'll try and then merge.

File: test/tools/jsontests/BlockChainTests.cpp (hunk 1 above)
Comment (pirapira) [inline test/tools/jsontests/BlockChainTests.cpp]: yes.

**Comment by pirapira** [comment]
#4366 should fix "the missing 0x problem".  This one should be merged after that.

**Comment by pirapira** [comment]
or not.

**Pull request merged**